servers/rendering/rasterizer_rd/rasterizer_canvas_rd.cpp

```diff
@@ -455,7 +455,7 @@ void RasterizerCanvasRD::_render_item(RD::DrawListID p_draw_list, const Item *p_
 
 				//bind textures
 
-				_bind_canvas_texture(p_draw_list, rect->texture, current_filter, current_repeat, last_texture, push_constant, texpixel_size);
+				_bind_canvas_texture(p_draw_list, rect->texture, current_filter, (rect->flags & CANVAS_RECT_TILE) ? RenderingServer::CanvasItemTextureRepeat::CANVAS_ITEM_TEXTURE_REPEAT_ENABLED : current_repeat, last_texture, push_constant, texpixel_size);
```

Comment:
this is probably unnecesary and the tile flag should be deprecated as you can change tiling in other means.

Reply:
What should it be replaced with? Should repeat be a boolean in `CommandRect`?